### PR TITLE
Add Overview Sections for Community, Developer, and Infrastructure

### PR DESCRIPTION
**Pull request by tsande16**
This PR will add the overview README pages which serve a the introduction to the sections: Community, Developer, and Infrastructure.

### CODE REVIEW

**Comment by tsande16** [comment]
Once all section PRs of the developer documentation are merged, the TOC can be updated.

File: community/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+planned initiatives around user engagement, community building, system administration, and application enhancements.
+
+## Getting Involved
+
+PASS is an open-source community project, and there are many ways to contribute—whether by providing feedback, reporting
+bugs, or writing code. Community members are encouraged to:
+
+* Review the existing code base and documentation.
+* Test the [PASS demo](https://demo.eclipse-pass.org) instance to understand workflows and features.
+* Join discussions on Slack to ask questions or offer suggestions.
```
Comment (cshannon2) [inline community/README.md:16]: Should we link to an invite / join screen for the Slack channel?

**Review by cshannon2 — APPROVED** [review]
This looks good to me. The only feedback is highlighted in the review.

File: community/README.md (hunk 1 above)
Comment (tsande16) [reply community/README.md:16]: Good catch! I will add this.

**Pull request merged**